library.lua

```diff
@@ -255,6 +255,36 @@ function _.values(t)
   return out
 end
 
+function _.once(f)
+    _.expect('partial', 1, 'function', f)
+    local cache
+    return function(...)
+        if not cache then
+            cache = table.pack(f(...))
+        end
+        return unpack(cache,1,cache.n)
+    end
+end
+
+function _.memoize(f)
+    _.expect('partial', 1, 'function', f)
+    local cache = {}
+    return function(...)
+        local key = select(1, ... )
```

Comment:
Why does memoize ignore the rest of the arguments? No good.

Reply:
As stated before i was following the lodash docs. I guess making it memorize all args would be workable if you want it that way?

Reply:
We're not lodash. Yes, do that.